docs/static/transforming-data.asciidoc

```diff
@@ -450,7 +450,79 @@ For example:
     }
 --------------------------------------------------------------------------------
 
-<<plugins-filters-jdbc_streaming,jdbc_streaming>>::
+
+<<plugins-filters-jdbc_static,jdbc_static filter>>::
+
+Enriches events with data pre-loaded from a remote database.
++
+The following example fetches data from a remote database, caches it in a local
+database, and uses lookups to enhance events with data cached in the local
+database.
++
+[source,json]
+-----
+filter {
+  jdbc_static {
+    loaders => [
```

Comment:
I'm tweaking the language a bit here and adding callouts to the example. I think we should avoid saying DDL because it's not really DDL (not a CREATE statement, for example).